### PR TITLE
Create speakers-2019.json

### PR DESCRIPTION
**Pull request by DigitalEmbrace**

### CODE REVIEW

File: src/views/eternacon/speaker-2019.jso [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        {
+            "speaker": "Dr. Manu Prakash, Rebecca Konte",
+            "time": "July 19, 2019, 23:00:00 UTC",
+            "topic": "Foldscope Hands-on Workshop" 
+        }
+    ],
+    "Saturday, July 20": [
+        {
+            "speaker": "Organizers",
+            "time": "July 20, 2022, 16:00:00 UTC",
```
Comment (luxaritas) [inline src/views/eternacon/speaker-2019.jso:17]: Looks like years are incorrect?

File: src/views/eternacon/speaker-2019.jso [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            "topic": "LinearPartition: Linear Time Partition Function and Base Pair Probability Calculation for RNA Secondary Structure"
+        },
+        {
+            "time": "July 20, 2022, 19:00:00 UTC",
+            "topic": "Break"
+        },
+        {
+            "speaker": "Brourd",
+            "time": "July 20, 2022, 19:45:00 UTC",
+            "topic": "Eterna100"
```
[the remark this answers is not in the document]
Comment (DigitalEmbrace) [reply src/views/eternacon/speaker-2019.jso:42]: The post-conference survey uses the same title. I'm unable to access the videos.

[the remark this answers is not in the document]
Comment (luxaritas) [reply src/views/eternacon/speaker-2019.jso:42]: Yep, verified myself as well

**Review by luxaritas — APPROVED** [review]
LGTM, thanks!

**Pull request merged**